js/ai.js

```diff
@@ -87,6 +87,14 @@ function foldWillLose(game) {
     return potentialPoints >= WIN_POINTS && potentialPoints >= game.rivalPoints;
 };
 
+function passOrCheck(game) {
+    if (game.playerBet == 0) {
```

Comment:
```suggestion
    if (game.lastAction == LastAction.NONE_YET || game.lastAction == LastAction.USER_PASS) {
```
Maybe this is more semantically correct?